mandatory/3-function-output.js

```diff
@@ -1,18 +1,20 @@
-// Add comments to explain what this function does. You're meant to use Google!
+//The getRandomNumber function will return a random integer from 0 to 9.
+
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+// The combine2Words function will combine the two words and create a new string.
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
```

Comment:
That's right, the `combine2Words()` function will combine two strings with the help of the `concat()` method.